app/main.ipynb

```diff
@@ -56,33 +57,59 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "array = [0] * 1000\n",
+    "\n",
+    "sys.getsizeof(array)"
```

Comment:
You correctly created a Python list of zeros and calculated its size. However, ensure that you also create a NumPy array of zeros with minimal memory usage as specified in the task. Consider using `np.zeros` with an appropriate data type to minimize memory usage.